internal/config/validation.go

```diff
@@ -254,8 +367,53 @@ func (c *Config) translateMoreThanOneAttestationError(ut ut.Translator, fe valid
 	return t
 }
 
+func registerTranslateDiskTypeError(ut ut.Translator) error {
```

Comment:
I think this is fine because the error message already gives you a list of all allowed values.
Before, the URLs where part of the config yaml comments and we did not have an exhaustive list of allowed values in that comment.